app/javascript/dashboard/components/widgets/modal/WootKeyShortcutModal.vue

```diff
@@ -0,0 +1,175 @@
+              </span>
+              <span class="shortcut-key key">
+                {{ $t('KEYBOARD_SHORTCUTS.KEYS.FORWARD_SLASH_KEY') }}
+              </span>
+            </div>
+          </div>
+          <i class="ion-android-close modal--close" @click="$emit('close')"></i>
+        </div>
+
+        <div class="shortcut__wrap">
+          <div class="title-key__wrap">
+            <h6 class="sub-block-title">
+              {{ $t('KEYBOARD_SHORTCUTS.TITLE.OPEN_CONVERSATION') }}
+            </h6>
+            <div class="shortcut-key__wrap">
+              <div class="open-conversation__key">
+                <span class="shortcut-key">
+                  {{ $t('KEYBOARD_SHORTCUTS.KEYS.ALT_OR_OPTION_KEY') }}
+                </span>
+                <span class="shortcut-key key">
+                  J
+                </span>
+              </div>
+              <span class="shortcut-key">
+                {{ $t('KEYBOARD_SHORTCUTS.KEYS.ALT_OR_OPTION_KEY') }}
+              </span>
+              <span class="shortcut-key key">
+                K
+              </span>
+            </div>
+          </div>
+
+          <div class="title-key__wrap">
+            <h6 class="sub-block-title">
+              {{ $t('KEYBOARD_SHORTCUTS.TITLE.RESOLVE_AND_NEXT') }}
+            </h6>
+            <div class="shortcut-key__wrap">
+              <span class="shortcut-key">
+                {{ $t('KEYBOARD_SHORTCUTS.KEYS.COMMAND_KEY') }}
+              </span>
+              <span class="shortcut-key">
+                {{ $t('KEYBOARD_SHORTCUTS.KEYS.ALT_OR_OPTION_KEY') }}
+              </span>
+              <span class="shortcut-key key">
+                E
+              </span>
+            </div>
+          </div>
+          <div
+            v-for="shortcutKey in shortcutKeys"
+            :key="shortcutKey.id"
+            class="title-key__wrap"
+          >
+            <h6 class="sub-block-title">
+              {{ title(shortcutKey) }}
+            </h6>
+            <div class="shortcut-key__wrap">
+              <span class="shortcut-key">
+                {{ shortcutKey.firstkey }}
+              </span>
+              <span class="shortcut-key key">
+                {{ shortcutKey.secondKey }}
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import { mixin as clickaway } from 'vue-clickaway';
+import { SHORTCUT_KEYS } from './constants';
+
+export default {
+  mixins: [clickaway],
+  data() {
+    return {
+      shortcutKeys: SHORTCUT_KEYS,
+    };
+  },
+  methods: {
+    title(item) {
+      return this.$t(`KEYBOARD_SHORTCUTS.TITLE.${item.label}`);
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.modal-container {
+  width: fit-content;
+}
+
+.page-top-bar {
+  padding: var(--space-medium) var(--space-large) var(--space-large)
+    var(--space-large);
+}
+
+.title-wrap {
+  display: flex;
```

Comment:
Can we check if `display: flex` is necessary here and also for css written below ?

Reply:
Yes, I have checked, I think we needed `display: flex` here.